client/components/Review.jsx

```diff
@@ -22,25 +29,64 @@ export default class Review extends React.Component {
    return result
  }
 
+ expandDescription() {
+   console.log('expanded ', this.state.expanded)
+   this.setState({
+     expanded: !this.state.expanded
+   })
+ }
 
  render() {
    var review = this.props.review[0]
    var stars = this.generateStars(review.rating)
+   var description = `${stars}  ${review.text}`;
+
+   let component = null
```

Comment:
I'd use a more descriptive word for 'component', like truncReview or something similar.

Reply:
Agreed. Changing to reviewBody.